docs/hadoop-catalog-with-adls.md

```diff
@@ -0,0 +1,525 @@
+---
+title: "Hadoop catalog with ADLS"
+slug: /hadoop-catalog-with-adls
+date: 2025-01-03
+keyword: Hadoop catalog ADLS
+license: "This software is licensed under the Apache License version 2."
+---
+
+This document describes how to configure a Hadoop catalog with ADLS (Azure Blob Storage).
```

Comment:
ADLS is based on `Azure Blob Storage`, but not `Azure Blob Storage`

Reply:
Here, we support Azure Blob Storage, ADLS, and ADLS (v2), I used `Azure Blob Storage` for all, but there are no abbreviations for "Azure Blob Storage", so I used `ADLS` to stands for storage service provided by Azure. 

Is there a good term to describe all?

Reply:
seems ADLS is enough

Reply:
I use this sentences to replace it `ADLS (aka. Azure Blob Storage (ABS), or Azure Data Lake Storage (v2))`